### ISSUE TITLE
Multi-line strings inside string interpolation have their contents changed

### ISSUE THREAD

**Issue by munificent** [dart-lang/dart_style#711]
This is a rare one, but a bad one. With the recent change to format string interpolation, it may actually erroneously change the contents of a string.

```dart
var str = '''${"""
a
b
c
"""}''';
```

Is reformatted to:

```dart
var str = '''${""" a b c """}''';
```

The really bad part is that dartfmt's sanity checking to prevent that doesn't catch this, because the change is a whitespace change, but to a part of the user's program where whitespace is meaningful.
